sympy/printing/tests/test_pycode.py

```diff
@@ -27,6 +27,9 @@ def test_PythonCodePrinter():
     assert prntr.module_imports == {'math': {'pi'}}
     assert prntr.doprint(acos(x)) == 'math.acos(x)'
     assert prntr.doprint(Assignment(x, 2)) == 'x = 2'
+    assert prntr.doprint(Piecewise((1, Eq(x, 0)),
+                        (2, x>6))) == '((1) if (x == 0) else (2) if (x > 6) else None)'
+    assert prntr.doprint(Piecewise((2, Le(x, 0)), (3, Gt(x, 0)))) == '((2) if (x <= 0) else (3))'
```

Comment:
This assertion is a bit confusing because Piecewise evaluates, removing the condition Gt(x, 0) before it's printed. Some of the tests above use evaluate=False option of Piecewise to prevent that, I think the same should be done here.